cbirrt2/cbirrtparameters.h

```diff
@@ -90,12 +90,17 @@ class CBirrtParameters : public PlannerBase::PlannerParameters
         if( !PlannerParameters::serialize(O) )
             return false;
 
+        std::streamsize old_precision = O.precision();
+        O.precision(17);

```

Comment:
Fixed, although +2 is necessary for exact roundtrips through strings (see http://stackoverflow.com/a/16941784/5228520)